src/proto/breakpoint.proto

```diff
@@ -17,6 +17,13 @@ message Breakpoint {
   repeated Variable evaluated_expressions = 10;
   Status status = 11;
   bool log_point = 12;
+  string log_message_format = 13;
+  enum LogLevel {
+    INFO = 0;
+    WARNING = 1;
+    ERR = 2;
```

Comment:
nit: Maybe ERROR? unless the debugger uses ERR

Reply:
I wasn't able to compile in C++ when I use ERROR so that's why I used ERR.